web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -374,7 +403,7 @@ const MemberSelectorDialog = ({ open, selectedMembers, onClose, onSubmit }) => {
           </IconButton>
           <div className="toolbar-title-container">
             <Typography className="toolbar-title" variant="h5">
-              Select Members
+              Select {memberDescriptor ?? 'Members'}
```

Comment:
Maybe 'Members' should be the default value of the `memberDescriptor` prop.

Reply:
Not sure why I didn't think of that. Nice clean-up. Implemented.